### PR TITLE
fix: Removed prechecks for existing accounts.

### LINKED ISSUE #2039: Remove pre-checks for non-existing entities
### Problem

In order to run full ethereum equivalence, we should be able to call non-existing addresses/contracts and these calls should result in SUCCESS with gas-charged as expected (21k base + N for the data field).

### Solution

Remove these pre-checks for non-existing entities:
- for the `from` param - [here](https://github.com/hashgraph/hedera-json-rpc-relay/blob/main/packages/relay/src/lib/eth.ts#L1587)
- for the `to` param - [here](https://github.com/hashgraph/hedera-json-rpc-relay/blob/main/packages/relay/src/lib/eth.ts#L1599)

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by ebadiere**
Removed prechecks for non-existing entities.

**Related issue(s)**:

Fixes #2039 

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: packages/relay/src/formatters.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@
  *
  * Hedera JSON RPC Relay
  *
- * Copyright (C) 2023 Hedera Hashgraph, LLC
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
```
Comment (AlfredoG87) [inline packages/relay/src/formatters.ts:5]:
```suggestion
 * Copyright (C) 2023-2024 Hedera Hashgraph, LLC
```

File: packages/relay/src/lib/constants.ts [hunk 2]
```diff
@@ -2,7 +2,7 @@
  *
  * Hedera JSON RPC Relay
  *
- * Copyright (C) 2023 Hedera Hashgraph, LLC
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
```
Comment (AlfredoG87) [inline packages/relay/src/lib/constants.ts:5]:
```suggestion
 * Copyright (C) 2023-2024 Hedera Hashgraph, LLC
```

File: packages/relay/tests/lib/formatters.spec.ts [hunk 3]
```diff
@@ -2,7 +2,7 @@
  *
  * Hedera JSON RPC Relay
  *
- * Copyright (C) 2023 Hedera Hashgraph, LLC
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
```
Comment (AlfredoG87) [inline packages/relay/tests/lib/formatters.spec.ts:5]:
```suggestion
 * Copyright (C) 2023-2024 Hedera Hashgraph, LLC
```

**Review by AlfredoG87 — COMMENTED** [review]
Looking good, just a bunch of nits regarding the year of the licence, but, not sure how we will end up deciding, if a range or a single year (latest = 2024)
Let me know!

File: packages/relay/src/lib/constants.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (ebadiere) [reply packages/relay/src/lib/constants.ts]: Done.

File: packages/relay/src/formatters.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (ebadiere) [reply packages/relay/src/formatters.ts]: Done.

File: packages/relay/src/lib/eth.ts [hunk 4]
```diff
@@ -2,7 +2,7 @@
  *
  * Hedera JSON RPC Relay
  *
- * Copyright (C) 2023 Hedera Hashgraph, LLC
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
```
[the remark this answers is not in the document]
Comment (ebadiere) [reply packages/relay/src/lib/eth.ts]: Done.

File: packages/relay/tests/lib/formatters.spec.ts (hunk 3 above)
Comment (ebadiere) [reply packages/relay/tests/lib/formatters.spec.ts:5]: Done.

File: packages/relay/tests/lib/eth/eth_call.spec.ts [hunk 5]
```diff
@@ -2,7 +2,7 @@
  *
  * Hedera JSON RPC Relay
  *
- * Copyright (C) 2023 Hedera Hashgraph, LLC
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
```
[the remark this answers is not in the document]
Comment (ebadiere) [reply packages/relay/tests/lib/eth/eth_call.spec.ts]: Done.

**Review by AlfredoG87 — APPROVED** [review]
LGTM 👍

**Pull request merged**